lib/aws-record/record/item_operations.rb

```diff
@@ -84,7 +84,24 @@ def save(opts = {})
 
       # Deletes the item instance that matches the key values of this item
       # instance in Amazon DynamoDB.
+      #
+      # @example Usage Example
+      #   class MyModel
+      #     include Aws::Record
+      #     integer_attr :id,   hash_key: true
+      #     string_attr  :name, range_key: true
+      #   end
+      #
+      #   model = MyModel.new(id: 4, name: "John")
+      #   model.name  # => "John"
+      #   model.save   
       # 
+      #   model.assign_attributes(id: 7, name: "Janeß")
+      #   model.id         # => 7
+      #   model.name       # => "Jane"
+      #   model.persisted? # => false
```

Comment:
Wait, is this true? This should be a `true` value if I understand `persisted?` properly, and per the ActiveRecord documentation of the same. Is this a mistake?

Reply:
You might be thinking of the `#dirty?` method, which would be false after saving, and true again after this operation.

Reply:
Oh I get this, we changed the key. A better example would NOT change the key, rather some non-key attributes.